README.md

```diff
@@ -29,25 +29,31 @@ Bump tries to identify your file specification and format automatically. You can
 * `asyncapi/v2/json`
 * `asyncapi/v2/yaml`
 
-The doc `id` and `token` options used below can be found in your documentation settings page on https://bump.sh. Note that you can replace these options by environments variables: `--id` can be replaced by `BUMP_ID`, and `--token` can by replaced by `BUMP_TOKEN`. This is useful to keep your private token secret.
+`doc` and `token` options used below can be found in your documentation settings page on https://bump.sh. Note that you can replace the token option by an environment variable, to keep it secret: `--token` can by replaced by `BUMP_TOKEN`.
 
 ### Preview
 
 You can preview your documentation by calling the `preview` command. A temporary preview will be created, with a unique URL. This preview will be available for 30 minutes. You don't need any credentials to use this command.
 
+Preview a documentation:
```

Comment:
Is it countable? If not what about saying: `Preview documentation:`?

Reply:
If we use this form here, why using the countable form elsewhere? (like "create a documentation").

Do you think we should change this for the whole README file?

Reply:
Can be treated as it is finally so let's carry on that way.